src/stripe/stripe-service.ts

```diff
@@ -155,36 +155,6 @@ export class StripeService {
             ".Input--invalid": {
               boxShadow: "none",
             },
-            ".TermsText": {
-              fontSize: textStyles.captionDefault[viewport].fontSize,
-              lineHeight: textStyles.captionDefault[viewport].lineHeight,
-            },
-            ".Tab": {
```

Comment:
Aren't Tabs used when we have more than one payment method?

Reply:
With the addition of the `ExpressCheckoutElement` we are relegated to only display the `card` method, at least as of now. But if we prefer to leave all these for a potential future I can just close the PR.

Reply:
Sorry I missed this bit, if we are 100% sure that other payment methods won't pop up it's fine (and actually better) to remove this unused CSS.

Reply:
Yes, we are only using "card" as payment method, and the wallets are deactivated implicitly in the `PaymentElement` given that we are using `ExpressCheckoutElement` in the same Stripe element group.